src/Components/Card/Card.jsx

```diff
@@ -1,16 +1,13 @@
 import { Link } from "react-router-dom";
 import randomImage from "../../assets/images/random.jpg";
 import "./Card.css";
-export function Card({data}) {
+export function Card({ data }) {
   return (
```

Comment:
destructure this data here in the component than use it.

`const {_id,description,category}= data`

Reply:
Better to use this way to avoid render failure:

const category = data?.category;